apis/management.cattle.io/v3/backup_types.go

```diff
@@ -62,6 +62,10 @@ type EtcdBackupSpec struct {
 	Filename string `yaml:"filename" json:"filename,omitempty" norman:"noupdate"`
 	// backupConfig
 	BackupConfig BackupConfig `yaml:",omitempty" json:"backupConfig,omitempty" norman:"noupdate"`
+	// version of k8s in the backup pulled from rke config
```

Comment:
Both fields need to be moved to status, spec should only be use for fields intended for a user to set.